src/api/Search.js

```diff
@@ -212,4 +212,102 @@ export default class Search {
         });
     });
   }
+
+  addOrDeleteDocument(jsonRequest: JSON, callback: () => void): Promise<any> {
```

Comment:
Should this be `addOrDeleteComment` or is this endpoint more generic than that?

Reply:
It is more generic.. you can add or delete any document by specifying the table field.